src/ImageSharp/PixelFormats/PixelOperations{TPixel}.cs

```diff
@@ -32,7 +31,7 @@ public partial class PixelOperations<TPixel>
     /// Gets the pixel type info for the given <typeparamref name="TPixel"/>.
     /// </summary>
     /// <returns>The <see cref="PixelTypeInfo"/>.</returns>
-    public virtual PixelTypeInfo GetPixelTypeInfo() => LazyInfo.Value;
+    public static PixelTypeInfo GetPixelTypeInfo() => TPixel.GetPixelTypeInfo();
```

Comment:
I'm assuming this is the reason you added the exclusion to the ruleset? 

Is there an advantage to making this static? If so, I'd rather the exclusion was in the source so we are forced to consider introduction of static fields elsewhere.